### PR TITLE
PCW documentation

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**
This is the documentation about Cloud Watch.
To make it easier to review please look on the same document in Gdrive https://docs.google.com/document/d/1l--r_HVZNKDKBZwtD3SEz_IjEV3Fn-Xn4eg-EzFvD4o/edit

This documentation is already reviewed by @fviale

### CODE REVIEW

**Comment by ShatalovYaroslav** [comment]
@mboussaa I applied all your comments. Could you review it again?

File: src/docs/PCW/PCWUserGuide.adoc [hunk 1]
```diff
@@ -223,8 +223,9 @@ Each monitoring type has an implementation of a Polling class. The polling param
         <kpi>#usageCombinedCPUMetric</kpi>
     </kpis>
 -----------------
-The part above describes a polling configuration for Monitoring physical metrics. This  particular example is for monitoring CPU. +
-The polling and calm periods correspond to the frequency of collecting the monitoring metrics. In our example, the class is *Jmx_Metrics*. This implementation takes two parameters: the jmx node url and the jmx credentials. The actual metric to monitor is represented by the *_<kpi>_* tag. Several metrics can be monitored at the same time. For our example after variables replacement the monitoring metric will be _"sigar:Type=CpuUsage Combined"._
+The code above describes a polling configuration for Monitoring physical metrics. This  particular example is for monitoring CPUs. +
+The polling and calm periods correspond to the frequency of collecting the monitoring metrics. The concrete set of the parameters rely on the implementation of a Polling class. In our example, the class is *Jmx_Metrics*. This implementation takes two parameters: the jmx node url and the jmx credentials.
```
Comment (mboussaa) [inline src/docs/PCW/PCWUserGuide.adoc]: The concrete set of parameters rely on...

File: src/docs/PCW/PCWUserGuide.adoc [hunk 2]
```diff
@@ -52,78 +52,78 @@ The core of rule processing is managed by the https://www.drools.org/[Drools Eng
...
+This behaviour is demonstrated in the next screenshot.
 
 image::PCW-activated-with-removed.PNG[align=center]
 
 [[_pcw_obtaining_url]]
 === Obtaining JMX node urls
-ProActive Resource Manager portal provides access to monitoring abilities. During the activation of a rule, a user should enter as rule parameter a JMX endpoint corresponding to the monitored resource (for example a ProActive Node). +
-In order to get this information for a Node, a user can go to *RM portal*, click with the right button on the Node which should be monitored and click `Copy JMX Endpoint`. Then a user can come back to *Cloud Watch* portal and paste the JMX url in corresponding field.
+ProActive Resource Manager portal provides access to monitoring capabilities. During the activation of a rule, a user should enter as rule parameter a JMX endpoint corresponding to the monitored resource (for example a ProActive Node). +
+In order to get this information from a Node, a user can go to the *RM portal*, clickon the right button of a Node which should be monitored and click `Copy JMX Endpoint`. Then, he can come back to *Cloud Watch* portal and paste the JMX url in the corresponding field.
```
Comment (mboussaa) [inline src/docs/PCW/PCWUserGuide.adoc]: click on

**Comment by ShatalovYaroslav** [comment]
@mboussaa thanks a lot for the review

**Pull request merged**